File: Datasets/HumanEval/HumanEval_Java/19_all.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;


class SortNumbers {
    /**
     * Input is a space-delimited string of numbers from 'zero' to 'nine'.
     * Valid choices are 'zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight' and 'nine'.
     * Return the string with numbers sorted from smallest to largest
     * >>> sort_numbers('three one five')
     * 'one three five'
     * >>> sort_numbers('two one six')
     * 'one two six'
     * @param numbers a space-delimited string of numbers from 'zero' to 'nine'
     * @return the string with numbers sorted from smallest to largest
     */
    public static String sortNumbers(String numbers) {

File: Cleared_Dataset/HumanEval_Py/19_all.py
from typing import List


def sort_numbers(numbers: str) -> str:
    """ Input is a space-delimited string of numerals from 'zero' to 'nine'.
    Valid choices are 'zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight' and 'nine'.
    Return the string with numbers sorted from smallest to largest.

    args:
        numbers: space-delimited string of numberals from 'zero' to 'nine'
    
    returns:
        string with numbers sorted from smallest to largest

    examples:
    >>> sort_numbers('three one five')
    'one three five'
    >>> sort_numbers('seven two nine')
    'two seven nine'
    """
